static/app/data/forms/accountPassword.tsx

```diff
@@ -14,7 +14,7 @@ const formGroups: JsonFormObject[] = [
         autoComplete: 'current-password',
         label: t('Current Password'),
         help: t('Your current password'),
-        placeholder: '',
+        placeholder: 'Your current password',
```

Comment:
oh, do we need to wrap these in `t()` or not?

Reply:
Oh, yes we do!